tests/cupy_tests/lib_tests/test_strided_tricks.py

```diff
@@ -25,10 +25,45 @@ def test_as_strided(self):
     def test_rolling_window(self, xp):
         a = testing.shaped_arange((3, 4), xp)
         a_rolling = rolling_window(a, 2, 0)
-
         return a_rolling
 
 
+class TestSlidingWindowView(unittest.TestCase):
```

Comment:
Could you add a test for 0-dim array?

Reply:
Thanks @asi1024 for your comment. Can you please clarify this further? 

I am not sure what you mean by 0-d array test?
Do you mean a scalar basically? If that is the case, the `sliding_window_view` should not work and should spit out an error?

I implemented these tests based on numpy tests for similar functionality and I could not find a 0-d array example there: 
(https://github.com/numpy/numpy/blob/8627dc96202899c01a75f46e3899e2f1118a293f/numpy/lib/tests/test_stride_tricks.py#L400-L430)

Reply:
Just for reference , I can wrote this : 

```
    @testing.numpy_cupy_array_equal()
    def test_0d(self, xp):
        # Create a 0-D array (scalar) for testing
        arr = xp.array(42)
        # Sliding window with window size 1
        window_size = 1 
        arr_view = xp.lib.stride_tricks.sliding_window_view(
            arr, window_size, 0)
        return arr_view    
      
```

Which raises an error for both numpy and cupy : AssertionError: Both cupy and numpy raise exceptions

So I think maybe you mean writing a test to make sure this fails with correct message?

Reply:
I added a test for this that checks the if `ValueError` is raised for 0-d array! Please let me know if this addressed your comment.